CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Try to keep listed changes to a concise bulleted list of simple explanations of changes. Aim for the amount of information needed so that readers can understand where they would look in the codebase to investigate the changes' implementation, or where they would look in the documentation to understand how to make use of the change in practice - better yet, link directly to the docs and provide detailed information there. Only elaborate if doing so is required to avoid breaking changes or experimental features from ruining someone's day.
 
 ## [Unreleased]
+* Add `BatchCheck` api endpoint [#2039](https://github.com/openfga/openfga/pull/2039)
```

Comment:
**Request:** Can you write a brief description of batch check and how to use it? Ex:

```
* `BatchCheck` API which allows multiple checks to be performed simultaneously. It can be used by caling the `/BatchCheck` endpoint and requires each check to assign a correlation ID. See [batch check docs](#) for more details. [#2039](https://github.com/openfga/openfga/pull/2039)
```